packages/aws-cdk-lib/aws-apigateway/test/method.test.ts

```diff
@@ -767,20 +769,137 @@ describe('method', () => {
 
     // WHEN
     new apigw.Method(stack, 'MethodAuthScopeUsed', {
-      httpMethod: 'POST',
+      httpMethod: 'OPTIONS',
       resource: api.root,
       options: {
         apiKeyRequired: true,
+        authorizationType: apigw.AuthorizationType.NONE,
         authorizationScopes: ['MethodAuthScope'],
       },
     });
 
     // THEN
-    Template.fromStack(stack).hasResourceProperties('AWS::ApiGateway::Method', {
-      ApiKeyRequired: true,
-      AuthorizationScopes: ['MethodAuthScope'],
+    expect(() => Template.fromStack(stack).hasResourceProperties('AWS::ApiGateway::Method', {
+      AuthorizationScopes: ['DefaultAuth'],
+    })).toThrow(/Template has 1 resources with type AWS::ApiGateway::Method, but none match as expected./);
+
+  });
+
+  test('Method options Auth Scopes should be None when Auth Type from Method is IAM', () => {
+    // GIVEN
+    const stack = new cdk.Stack();
+    const api = new apigw.RestApi(stack, 'test-api', {
+      cloudWatchRole: false,
+      deploy: false,
+      defaultMethodOptions: {
+        authorizationType: apigw.AuthorizationType.IAM,
+        authorizationScopes: ['DefaultAuth'],
+      },
+    });
+
+    // WHEN
+    new apigw.Method(stack, 'MethodAuthScopeUsed', {
+      httpMethod: 'OPTIONS',
+      resource: api.root,
+      options: {
+        apiKeyRequired: true,
+        authorizationType: apigw.AuthorizationType.NONE,
```

Comment:
A clarifying question, does this not override the `authorizationType` set in the `defaultMethodOptions`

Reply:
Updated the test, no longer overriding.